src/test/kotlin/nl/info/zac/authentication/UserPrincipalFilterTest.kt

```diff
@@ -124,7 +124,7 @@ class UserPrincipalFilterTest : BehaviorSpec({
                 "behandelaar",
                 "domein_elk_zaaktype"
```

Comment:
```suggestion
                "fakeRole1",
                "fakeRole2"
```

I would not use `domein_elk_zaakype` etc here. That is a role name from the old IAM world. Has nothing to do with PABC / new IAM world.